webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js

```diff
@@ -452,16 +453,20 @@ export const ErrataTab = () => {
                     />
                   </Tr>
                   <Tr key="child_row" isExpanded={isExpanded}>
-                    <Td colSpan={3}>
-                      <ExpandableRowContent>
-                        <ErratumExpansionContents erratum={erratum} />
-                      </ExpandableRowContent>
-                    </Td>
-                    <Td colSpan={4}>
-                      <ExpandableRowContent>
-                        <ErratumExpansionDetail erratum={erratum} />
-                      </ExpandableRowContent>
-                    </Td>
+                    {isExpanded && (
```

Comment:
The conditional rendering here was by far the most helpful change of all. Not having to render invisible table rows just makes sense :smile:

Reply:
This is good, and we should probably do that [here](https://github.com/Katello/katello/blob/master/webpack/scenes/ContentViews/Table/ContentViewsTable.js#L241) 
And any other places we use expandable content.

Reply:
Agree. But I feel like only tables that use `useBulkSelect` are in scope of this PR, right?

Reply:
You can see performance issues for both.  If the fix applied here is only affecting useBulkSelect's behaviour than that's totally reasonable.